test/plugin/base_test.rb

```diff
@@ -338,6 +338,81 @@ def test_structured_payload_log
     end
   end
 
+  def test_stackdriver_trace_format_compliant_trace
+    full_stackdriver_trace = \
+      'projects/some-proj/traces/1234567890abcdef1234567890abcdef'
+
+    [
+      {
+        config: APPLICATION_DEFAULT_CONFIG
+      },
+      {
+        config: 'enable_stackdriver_trace_autoformat true'
```

Comment:
Please extract this and the next config string into constants, and define them in terms of `APPLICATION_DEFAULT_CONFIG`.

Reply:
+1. Similar to https://github.com/GoogleCloudPlatform/fluent-plugin-google-cloud/blob/master/test/plugin/constants.rb#L235.